DEVELOPMENT.md

```diff
@@ -130,3 +130,19 @@ $ bin/yard server --reload
 
 Then navigate to `localhost:8808` to view the rendered docs.
 
+### Publish documentation to rspec.github.io
```

Comment:
Does it make sense to mention that rspec.github.io is the same as rspec.info?

Reply:
Yep. Good idea

Reply:
https://github.com/rspec/rspec-rails/compare/d48db33e77663114ee58223f96bdd94d939b0836..56f6c597429f8d7319d9ebfff3fb2505fd970dd7